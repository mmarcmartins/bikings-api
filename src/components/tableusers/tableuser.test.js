import React from "react";
import { shallow, mount } from "enzyme";
import TableUsers from "./tableusers";

describe("Table Users", () => {
  it("Table user should render correctly if users is null", () => {
    const component = shallow(<TableUsers />);
    expect(component).toMatchSnapshot();
  });
});
