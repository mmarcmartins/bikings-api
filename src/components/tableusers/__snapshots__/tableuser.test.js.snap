// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table Users Table user should render correctly 1`] = `
<div
  className="table-container"
>
  <table
    className="table-users"
  >
    <thead>
      <tr>
        <th>
          Username
        </th>
        <th>
          Name
        </th>
        <th>
          Email
        </th>
        <th>
          City
        </th>
        <th>
          Ride in group
        </th>
        <th>
          Day of the week
        </th>
        <th>
          Posts
        </th>
        <th>
          Albums
        </th>
        <th>
          Photos
        </th>
        <th />
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;

exports[`Table Users Table user should render correctly if users is null 1`] = `
<div
  className="table-container"
>
  <table
    className="table-users"
  >
    <thead>
      <tr>
        <th>
          Username
        </th>
        <th>
          Name
        </th>
        <th>
          Email
        </th>
        <th>
          City
        </th>
        <th>
          Ride in group
        </th>
        <th>
          Day of the week
        </th>
        <th>
          Posts
        </th>
        <th>
          Albums
        </th>
        <th>
          Photos
        </th>
        <th />
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;

exports[`Table Users should render correctly in "debug" mode 1`] = `
<div
  className="table-container"
>
  <table
    className="table-users"
  >
    <thead>
      <tr>
        <th>
          Username
        </th>
        <th>
          Name
        </th>
        <th>
          Email
        </th>
        <th>
          City
        </th>
        <th>
          Ride in group
        </th>
        <th>
          Day of the week
        </th>
        <th>
          Posts
        </th>
        <th>
          Albums
        </th>
        <th>
          Photos
        </th>
        <th />
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
